vertx-web/src/main/asciidoc/index.adoc

```diff
@@ -88,6 +88,15 @@ dependencies {
 }
 ----
 
+=== Development mode
+
+Vert.x Web by default operates in production mode.
+You can switch the development mode by assigning the `dev` value to either:
+
+* the `VERTXWEB_ENVIRONMENT` environment variable, or
+* the `vertxweb.environment` system property
+
+As an example, in development mode, template engines and static handlers caches are disabled.
```

Comment:
I think it is better to mention precisely what the dev mode does and not say "as example".

Reply:
also explains what are the side effects and what the user can expect with dev mode

Reply:
Do you mean here or in the relevant handler sections? Here it would not be great I think, because hard to maintain.

Reply:
@vietj ping

Reply:
I think here, it is important that all the effects of setting this property has and what the user can expected from this

Reply:
Done